src/util.cpp

```diff
@@ -488,6 +488,12 @@ namespace Sass {
 
       int cp = utf8::next(it, end);
 
+      // in case of \r, check if the next in sequence
+      // is \n and then advance the iterator.
+      if (cp == '\r' && it < end && utf8::peek_next(it, end) == '\n') {
```

Comment:
@xzyfer, seems like in some cases, we do need to `push_back()` the carriage return to quoted string: https://travis-ci.org/sass/libsass/jobs/99452071#L1108.